i18n/ext/discord.i18n.js

```diff
@@ -0,0 +1,364 @@
+import { genI18nMessages } from '~/utils/i18n.utils'
+
+export default genI18nMessages({
+    'en-us': {
+        title: 'PyCon TW Discord Guideline',
+        pageAbstract:
+            'If you have not installed Discord or joined Python Taiwan Discord server, follow Setting up Discord and setup your Discord first.',
+        tutorialsHeader: 'Tutorials',
+        stepOne: '1. Register',
+        stepOneDescription:
+            'Users can find a #registration-desk channel under PYCON TW 2021 category. There is a PyCon TW RegBot in this channel to help you register to your roles.',
+        registerAction: {
+            action: 'Enter !register [TOKEN] in this channel to register',
+            note:
+                '(If you want to get your own Token, please click the link in the letter or use Opass.)',
```

Comment:
We will probably not integrate with OPass this year

```suggestion
                '(If you want to get your own Token, please click the link in the letter.)',
```